src/odemis/acq/stream/_live.py

```diff
@@ -77,6 +77,32 @@ def _find_metadata(self, md):
 
         return simpl_md
 
+    def getPixelCoordinates(self, p_pos):
+        """
+        Translate physical coordinates into data pixel coordinates
+        Args:
+            p_pos: the position in physical coordinates
+
+        Returns: the position in pixel coordinates
+
+        """
+        if self.raw:
+            md = self.raw[0].metadata
+            pxs = md.get(model.MD_PIXEL_SIZE, (1e-6, 1e-6))
+            # (l, t, r, b): bounding box
+            bounding_box = self.getBoundingBox()
+            # Calculate the distance to the left bottom in physical units
+            if bounding_box[0] <= p_pos[0] <= bounding_box[2] and bounding_box[1] <= p_pos[1] <= bounding_box[3]:
+                dist = (p_pos[0] - bounding_box[0],
+                        - (p_pos[1] - bounding_box[3]))
+                return int(dist[0] / pxs[0]), int(dist[1] / pxs[1])
```

Comment:
Sorry, I realise just now that this is not going to be sufficient for data which is rotated. On the SECOM, the images can have MD_SHEAR and MD_ROTATION. In getBoundingBox, we incorrectly don't take them into account. As they are usually small, it's probably not a big deal. However, to know precisely which pixel corresponds a specific position, you *must* take them into account. You can use the (new) 20180703-MPI-pond-01.ome.tiff in our test data set for checking. I'd also suggest to play with the "manual overlay" plugin to increase the rotation and shear, so that it's even more obvious.

I *think* that the position of the pixels in physical coordinates is computed using a transformation matrix applied with the center of the image as referential, with rotation * shear*scale + translation. The exact way is coded in the gui.comp.canvas.draw() function. It might be best to first confirm the exact formula by trying to plot the top left corner (or any corner) of the image using an overlay and only the metadata. Once this matches the top-left corner, you "just" need to inverse the formula to compute the physical coordinates in the pixel coordinates.